test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Expression.Tests.ps1

```diff
@@ -6,25 +6,13 @@ Describe "Invoke-Expression" -Tags "CI" {
 	    (Invoke-Expression -command "echo pestertest1") | Should be "pestertest1"
 	}
 
-	It "Should return the echoed text for the alias" {
-	    (iex -command "echo pestertest2") | Should Be "pestertest2"
-	}
```

Comment:
Same here. It's not testing whether the alias `iex` eixsts, but testing the functionality of `iex`. So it shouldn't be removed.